### DISCUSSION TITLE
freq and  domainstats from a another machine

### DISCUSSION THREAD

**Discussion by l0x2a** [Security-Onion-Solutions/securityonion#1649] [Unsupported Versions] [answered]
Hey Onion-People,
i've just discovered that you're supporting freq and domainstats. I have these tools already running on another server and I'm wondering if I can use these from my brandnew SO v2.3.1 Installation (ISO Install v2.3, then update via soup) without launching the corresponding docker containers on my SO? And if that is possible ... how do I configure it correctly without sprinkle 'salt' in someone's wounds?
Cheers, Sam

**Comment by weslambert** [comment, marked as answer]
I wouldn't recommend connecting it to an external instance unless you are constraining the processing to a separate pipeline, as it could cause latency to the standard pipeline(s), especially if there are network connection issues between the nodes.
